Sources/CasePathsMacros/CasePathableMacro.swift

```diff
@@ -422,3 +423,16 @@ final class SelfRewriter: SyntaxRewriter {
     return super.visit(node.with(\.name, self.selfEquivalent))
   }
 }
+
+extension StringProtocol {
```

Comment:
I wish Swift would have a built-in equivalent to `trimmingPrefix(while)`, but this will do